src/platform/web/ui/session/rightpanel/MemberTileView.js

```diff
@@ -0,0 +1,11 @@
+import {TemplateView} from "../../general/TemplateView.js";
+import {AvatarView} from "../../avatar.js";
+
+export class MemberTileView extends TemplateView {
+    render(t, vm) {
+        return t.div({ className: "MemberTileView" }, [
```

Comment:
the root element should probably be an `<li>` here as it's mounted within an `<ul>`.